### DISCUSSION TITLE
Debian Buster 64 two nics is working!

### DISCUSSION THREAD

**Discussion by rkwesk** [ltsp/ltsp#166] [Q&A]
Clean very basic install on old bios pc. I'm writing this from the client through epoptes at the server's keyboard.  My test bed has an old server with an onboard gigabit nic connected to an old disused router and a client with 100 bit nic. There are no other clients!!

Still it seems possible to write this comment and hopefully to send it.

I will open a new issue in ltsp/ltsp/issues to discuss more technical stuff.

Richard

**Comment by alkisg** [comment]
Awesome!

I'll now work on `ltsp ipxe`, to allow it to be completely generated from ltsp.conf parameters, even the menu text or client kernel cmdline parameters.

**Comment by rkwesk** [comment]
Another success with a clean install of Debian Buster with two nics. This time the server is a laptop with one gigabit interface for the lan and (just to see if could work) the wireless interface is used for the wan.

Again I am writing this from the laptop keyboard through epoptes which is controlling the client.

Again I use the mate desktop and network manager is controlling both interfaces.

Richard
